### ISSUE TITLE
Bookmark toolbar is behind every page and doesn't emerge with side tabs

### ISSUE THREAD

**Issue by AliBar99** [Naezr/ShyFox#210]
Hi
I hit this issue and I don't know how to fix it. My bookmark toolbar, above the side tab list, is hidden under every page I open. I restarted Firefox's setting and even tried reinstalling.

![Screenshot 2024-11-30 133321](https://github.com/user-attachments/assets/c55411b4-7208-4c2d-b464-9a7cdf16e6e1)

**Comment by LaCombine** [comment]
Happened to me yesterday after updating to Firefox 133.0. Fixed it by disabling styles, i think ShyFox needs an update before enabling styles again 
![20241130_111731](https://github.com/user-attachments/assets/6ac9b87e-24e6-4fa2-982a-96c3c9b5cfb4)

**Comment by AliBar99** [comment]
Still not working. Is there any setting to modify the sidebery or other extensions manually?

**Comment by LaCombine** [comment]
Did you restart Firefox? I didn't change anything else tbh, so if it doesn't work that's a problem. You're using FF 133.0 too?

**Comment by AliBar99** [comment]
Yeah first I've cleared the FF cache then restart every FF setting and followed the shyfox install instructions (I'm using v. 133.0).

**Comment by LaCombine** [comment]
You made a fresh new installation? Did you use it before and it worked well or is it your first install? Maybe if you have a backup it could help you

**Comment by AliBar99** [comment]
Yeah I did. I am using sidebery for months and it worked properly, until the new FF update.
Unfortunately I don't have a backup.

**Comment by Sneaky1000** [comment]
I'm having the same issue. Definitely seems like a styling conflict but I'm not sure what/where I could look to fix this.

**Comment by Sneaky1000** [comment]
Has anyone found a workaround or temporary solution to this?

**Comment by Sneaky1000** [comment]
If anyone is still having this issue, try my patch: #243 

I have a very custom version of this theme at this point, but that change should fix the toolbar issue. If you are still having issues after that fix, send me a screenshot and I'll install the theme fresh to see if I can fix it.
